README.md

```diff
@@ -1 +1,118 @@
 # xtractmime
+
+`xtractmime` is a [BSD-licensed](https://opensource.org/licenses/BSD-3-Clause)
+Python 3.6+ implementation of the [MIME Sniffing
+Standard](https://mimesniff.spec.whatwg.org/).
+
+Install from [`PyPI`](https://pypi.python.org/pypi/xtractmime):
+
+```
+pip install xtractmime
+```
+
+---
+
+## Basic usage
+
+Below mentioned are some simple examples of using `xtractmime.extract_mime`:
+
+```python
+>>> from xtractmime import extract_mime
+>>> body = b"Sample text content"
+>>> extract_mime(body)
+b'text/plain'
+>>>
+```
+
+```python
+>>> from xtractmime import extract_mime
+>>> content_types = (b'text/html',)
+>>> body = b''
+>>> extract_mime(body, content_types=content_types)
+b'text/html'
+>>> 
+```
+
+Additional functionality to check if a mime type belongs to a specific mime type group using 
+methods mentioned in `xtractmime.mimegroups`
+
+```python
+>>> from xtractmime.mimegroups import *
+>>> mime_type = b'text/html'
+>>> is_html_mime_type(mime_type)
+True
+>>> is_image_mime_type(mime_type)
+False
+>>>
+```
```

Comment:
Do not suggest [bad practices](https://stackoverflow.com/q/2386714) in the documentation :slightly_smiling_face: 

````suggestion
```python
>>> from xtractmime.mimegroups import is_html_mime_type, is_image_mime_type
>>> mime_type = b'text/html'
>>> is_html_mime_type(mime_type)
True
>>> is_image_mime_type(mime_type)
False
```
````

Reply:
😮 sorry